src/handlers/completion.rs

```diff
@@ -153,7 +156,7 @@ async fn find_completions(
 ) -> Result<CompletionList, String> {
     let uri = params.text_document.uri;
     let pos = params.position.clone();
-    let name = get_ident_name(uri.clone(), params.position)?;
+    let name = get_ident_name(uri.clone(), params.position, 1)?;
```

Comment:
what is this magical number here? can this be pulled out into a const/var and named to add context?

Reply:
"==" | "!=" vs "." | ">" | ":"